.github/workflows/Update_Dev_Docker_Image.yml

```diff
@@ -0,0 +1,79 @@
+name: Update Docker Development Docker Images
+
+on:
+  schedule:
+    - cron: '0 0 * * 6'
+
+jobs:
+  build_push_dev_docker_images:
+    name: Build and Push Docker Development Images
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3.3.0
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2.5.0
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2.1.0
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Setup .npmrc for repository
+        run: |
+            echo -e "@tiptap-pro:registry=https://registry.tiptap.dev/\n//registry.tiptap.dev/:_authToken=${{ secrets.TIPTAP_TOKEN }}" > .npmrc
+
+      - name: Build and Push Frontend to Docker Container Registry
+        uses: docker/build-push-action@v4.0.0
+        with:
+          context: .
+          file: ./apps/app/Dockerfile.web
+          platforms: linux/amd64
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/plane-frontend:develop
```

Comment:
We cannot hardcode the tag as `develop`. Per branch we will have docker hub repo - e.g. 
- makeplane/plane-frontend-[branch-name]:[run_id]
- makeplane/plane-space-[branch-name]:[run_id]
- makeplane/plane-backend-[branch-name]:[run_id]
- makeplane/plane-proxy-[branch-name]:[run_id]